spec/cucumber/formatter/html_spec.rb

```diff
@@ -567,5 +567,24 @@ module Formatter
         end
       end
     end
+
+    describe Html::SnippetExtractor do
+      describe '#snippet' do
+        let(:instance) { Html::SnippetExtractor.new }
+
+        context 'when the error string is frozen' do
+          before { Html::SnippetExtractor.class_variable_set(:@@converter, Html::SnippetExtractor::NullConverter.new) }
+
+          context 'the error line doesnt end with a colon and a line number' do
+            let(:error_line) { 'path/to/file'.freeze }
+            let(:error) { [error_line, nil] }
+
+            it do
```

Comment:
Perhaps this `it` could be described, so that a rspec output using “-f documentation” has more complete sentences?

Reply:
Yep, I'll flesh this out. Thanks!